client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -42,6 +43,8 @@ const Header: React.FunctionComponent = () => {
 	// CreateSite step clears state before redirecting, don't show the default text in this case
 	const siteTitleDefault = 'CreateSite' === currentStep ? '' : __( 'Start your website' );
 
+	const homeLink = '/';
```

Comment:
> in logged-in state check user is routed to thier primary site: e.g. /home/myprimarysite.wordpress.com

So in order to do this we'd have to grab `primary_blog` id from the [User store](https://github.com/Automattic/wp-calypso/tree/master/packages/data-stores/src/user).

Good news is that I think that piece of information is  available in the [API response](https://github.com/Automattic/wp-calypso/blob/410b3d6ffa089d66c8f571747334eafbc0233895/packages/data-stores/src/user/resolvers.ts#L27). 

We can check if that's true and, if so, add it to the [CurrentUser](https://github.com/Automattic/wp-calypso/blob/410b3d6ffa089d66c8f571747334eafbc0233895/packages/data-stores/src/user/types.ts#L6) type.

If it all turns out to be too much trouble, I think we could merge what you've got (since it's better than no link) and do the rest in a follow up PR. 

👍

Reply:
I think my statement about  `/home/myprimarysite.wordpress.com` could be a little confusing.

For a logged-in user: 
There is a redirect elsewhere (not to do with changes in this PR) that redirects from `/` to `/home/myprimarysite.wordpress.com`.  So for testing purposes, a tester will see that when logged-in and on /new, clicking the (W) icon will land the tester on `/home/myprimarysite.wordpress.com`.  😄